pkg/registry/error.go

```diff
@@ -68,3 +69,12 @@ func IsRepositoryNotFound(err error) bool {
 		return false
 	}
 }
+
+var noMorePagesError = &microerror.Error{
+	Kind: "noMorePagesError",
+}
+
+// IsNoMorePages asserts noMorePagesError.
+func IsNoMorePages(err error) bool {
```

Comment:
This error should come before `IsRepositoryNotFound` - `RepositoryNotFound` is after `NoMorePages` in the alphabet #nitpick